services/motion/builtin/builtin.go

```diff
@@ -51,7 +51,11 @@ func init() {
 		})
 }
 
-const builtinOpLabel = "motion-service"
+const (
+	builtinOpLabel         = "motion-service"
+	defaultAngularVelocity = 60  // degrees per second
```

Comment:
[nit] lets add a comment here that this specifically pertains to Base movement.  This file deals with our Arm related endpoints too, so not having that context could cause

Reply:
Another option: we can put these defaults in `kinematicbase`, and here we can pass in `math.NaN` or `0`, which the wrapping function can replace with defaults if it sees.

Reply:
Also: these variables should, regardless of where they live, have units on the variable name, not just a comment. E.g. `defaultAngularVelocityDPS`

Reply:
I'm hoping that long term we can move towards an "options" style approach similar to what we do in `motionplan` for planners.  Until we have that in place I think doing what Kat has suggested is a good approach, since being explicit about what is being passed in is more readable.  

When we have options, ommitted values will be zero and we can add the defaults on the kinematics side when we read them